R/view_annotation_dictionary.R

```diff
@@ -0,0 +1,47 @@
+#' Browse ecocomDP's dictionary for annotating datasets and attributes
+#'
+#' @description  
+#'     View the collection of dataset- and attribute-level annotations
+#'     from existing ecocomDP datasets.
+#'
+#' @usage 
+#'     view_annotation_dictionary()
```

Comment:
Usage is automatically created when building the function doc. No need to listed here. Please remove these lines.